### PR TITLE
Set Server.version to Cheroot version instead of CherryPy version

### LINKED ISSUE #5: wsgi.Server reports CherryPy version
As I understand it, Cheroot brings a standalone server with seperate versioning (e.g. 5.1), but 
`cheroot.wsgi.Server.version` returns either "CherryPy/undefined" or "CherryPy/10.1.1", depending on the additional cherrypy package beeing installed or not.

### PR DESCRIPTION
**Pull request by mar10**
Fix #5

### CODE REVIEW

File: cheroot/server.py [hunk 1]
```diff
@@ -1130,7 +1130,8 @@ class HTTPServer(object):
     timeout = 10
     """The timeout in seconds for accepted connections (default 10)."""
 
-    version = 'CherryPy/' + cp_version
+    # version = 'CherryPy/' + cp_version
+    version = 'Cheroot/' + __version__
```
Comment (webknjaz) [inline cheroot/server.py]: I'm not sure about branding (whether it should begin with capital `C`, @jaraco should it?), but this looks good :)

**Comment by mar10** [comment]
What would be a good place to add tests?
Cheroot status says
> the test suite is currently disabled. Currently, the only way to validate the test
suite is to run it against CherryPy.

I als could not get the CherryPy tests to work:
```
(cp35) $ python setup.py test
/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'python_requires'
  warnings.warn(msg)
running test
(cp35) $ 
```

Running tox stops at this line
```
(cp35) $ tox
...
collecting 0 items
```
Maybe I am missing some dependencies?

**Comment by webknjaz** [comment]
Oh.. that's a tricky question. I'll merge this w/o tests. They aren't fully set up yet.
But in case you'll want to add them in the future, they'll go to https://github.com/cherrypy/cheroot/tree/master/cheroot/test

**Comment by jaraco** [comment]
Thanks both!

**Pull request merged**